src/handlers/googleAuthHandlers.ts

```diff
@@ -0,0 +1,34 @@
+import ElectronGoogleOAuth2 from 'electron-google-oauth2';
+import {
+  clientId, clientSecret, createClient, redirectUri, scopes, validateToken
+} from '@/backend/export/outputVendors/googleSheets/googleAuth';
+import { getAllSpreadsheets } from '@/backend/export/outputVendors/googleSheets/googleSheetsInternalAPI';
+import { createSpreadsheet } from '@/backend/export/outputVendors/googleSheets/googleSheets';
+
+export const googleLoginHandler = () => {
```

Comment:
I see there are almost no type definitions in the code.
It is very helpful (especially in a collaborative project) to have type definitions where possible. Please add for function parameters and return values of functions